lib/tasks/pkg.rake

```diff
@@ -23,6 +23,17 @@ namespace :pkg do
     ref = ENV['ref'] || 'HEAD'
     version = `git show #{ref}:VERSION`.chomp.chomp('-develop')
     raise "can't find VERSION from #{ref}" if version.empty?
-    `git archive --prefix=foreman-#{version}/ #{ref} | bzip2 -9 > pkg/foreman-#{version}.tar.bz2`
+    `git archive --prefix=foreman-#{version}/ #{ref} > pkg/foreman-#{version}.tar`
+
+    `npm install --no-optional --global-style true`
```

Comment:
TBH It would be ever more reliable imho if we switch to yarn, but I'm OK with doing it in two pr's..